2_process/src/recursive_fun.R

```diff
@@ -0,0 +1,56 @@
+library(targets)
```

Comment:
Are you planning to make this R script only for the function after you've got it working the way you want it? (remove the library and tar_make lines, etc.)

Reply:
Yes I am planning to make this R script just the function. Still in progress as I have not yet gotten to this part yet.